npm/css-src/sass/main.scss

```diff
@@ -14,3 +14,41 @@
   .hide {
   display: none;
 }
+
+.drag-and-drop {
+  margin-bottom: 40px;
+}
+
+.drag-and-drop__dropzone {
+  display: flex;
+  outline: 3px dashed #0b0c0c;
+  padding: 60px 15px;
+  text-align: center;
+  transition: outline-offset 0.1s ease-in-out, background-color 0.1s linear;
+}
+
+.drag-and-drop__dropzone [type=file] {
+  display: none;
+  left: 10.1em;
+  position: relative;
+}
+
+.drag-and-drop__dropzone p {
+  margin-bottom: 0;
+  margin-right: 10px;
+  padding-top: 7px;
+}
+
+.drag-and-drop__dropzone label {
+  display: inline-block;
+  margin-bottom: 0;
+  width: auto;
+}
+
+.file-warning-message {
+  margin-top: 20px;
```

Comment:
I'm not 100% sure of this, but the GOV.UK design system seems to follow the principle of adding margins to the bottom of elements to give the element below enough space. If you add margins to the top _and_ bottom of elements, it makes it harder to mix and match those elements in other contexts.

Is it possible to add a margin to the bottom of the drag-and-drop instead? It looks like that would be more consistent.

Reply:
Good point! Sure!